coreneuron/io/phase1.cpp

```diff
@@ -20,10 +21,24 @@ void Phase1::read_file(FileHandler& F) {
 
     this->output_gids = F.read_vector<int>(n_presyn);
     this->netcon_srcgids = F.read_vector<int>(n_netcon);
+    // For file mode transfer, it is not allowed that negative gids exist
+    // in different threads. So this->netcon_tids remains clear.
 
     F.close();
 }
 
+static void prsrctid(int thread_id, int n, int* srcgid, std::vector<int>& srctid) {
```

Comment:
function not referenced.

Reply:
Forgot that debugging helper was there. Will remove.